{
	"schema_version": "1.6.0",
	"id": "CVE-2021-38208",
	"modified": "2024-09-18T01:00:20Z",
	"published": "2021-08-08T20:15:07Z",
	"related": [
		"DLA-2689-1",
		"DLA-2690-1",
		"SUSE-SU-2022:1651-1",
		"SUSE-SU-2022:1668-1",
		"SUSE-SU-2022:1669-1",
		"SUSE-SU-2022:1676-1",
		"SUSE-SU-2022:1686-1",
		"SUSE-SU-2022:1687-1",
		"SUSE-SU-2022:2077-1",
		"SUSE-SU-2022:2082-1",
		"SUSE-SU-2022:2083-1",
		"SUSE-SU-2022:2103-1",
		"SUSE-SU-2022:2111-1",
		"USN-5050-1",
		"USN-5343-1"
	],
	"details": "net/nfc/llcp_sock.c in the Linux kernel before 5.12.10 allows local unprivileged users to cause a denial of service (NULL pointer dereference and BUG) by making a getsockname call after a certain type of failure of a bind call.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:11",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.10.46-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2021-38208.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:12",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.10.46-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2021-38208.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:13",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.10.46-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2021-38208.json"
			}
		}
	],
	"references": [
		{
			"type": "FIX",
			"url": "https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.12.10"
		},
		{
			"type": "WEB",
			"url": "http://www.openwall.com/lists/oss-security/2021/08/17/1"
		},
		{
			"type": "WEB",
			"url": "http://www.openwall.com/lists/oss-security/2021/08/17/2"
		},
		{
			"type": "WEB",
			"url": "http://www.openwall.com/lists/oss-security/2021/08/24/2"
		},
		{
			"type": "FIX",
			"url": "https://github.com/torvalds/linux/commit/4ac06a1e013cf5fdd963317ffd3b968560f33bba"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1992810"
		},
		{
			"type": "ADVISORY",
			"url": "https://security-tracker.debian.org/tracker/CVE-2021-38208"
		}
	]
}
